Makefile

```diff
@@ -77,21 +77,9 @@ DOCKER_RUN_DOCKER := $(DOCKER_FLAGS) "$(DOCKER_IMAGE)"
 
 default: binary
 
-all: build ## validate all checks, build linux binaries, run all tests\ncross build non-linux binaries and generate archives
+all: build ## validate all checks, build linux binaries, run all tests\nbuild non-linux binaries and generate archives
 	$(DOCKER_RUN_DOCKER) bash -c 'hack/validate/default && hack/make.sh'
 
-binary: build ## build the linux binaries
-	$(DOCKER_RUN_DOCKER) hack/make.sh binary
-
-build: bundles init-go-pkg-cache
```

Comment:
cleanup `\n` in comments (replace with space or real newline).

i reported this in your previous PR: https://github.com/docker/docker/pull/26389#discussion_r87918159